New @lit-labs/context feature is broken with Typescript

# Description

It looks like there is a publishing error with the first release of `@lit-labs/context`. The `index.d.ts` files in the `lib` directory which is not present.

```sh
❯ cat index.d.ts
/**
 * @license
 * Copyright 2021 Google LLC
 * SPDX-License-Identifier: BSD-3-Clause
 */
export { ContextCallback, ContextRequestEvent as ContextEvent, } from './lib/context-request-event.js';
export { ContextKey, ContextType, createContext } from './lib/context-key.js';
export { ContextConsumer } from './lib/controllers/context-consumer.js';
export { ContextProvider } from './lib/controllers/context-provider.js';
export { ContextRoot } from './lib/context-root.js';
export { contextProvider } from './lib/decorators/context-provider.js';
export { contextProvided } from './lib/decorators/context-provided.js';
//# sourceMappingURL=index.d.ts.map%                                                                                        

❯ ls
LICENSE         development/    index.d.ts.map  index.js.map    package.json
README.md       index.d.ts      index.js        node_modules/   src/
```

# Steps to Reproduce

1. Write this code

```ts
import { ContextProvider, createContext } from '@lit-labs/context'
```

2. See this tsc compiler output...

```
../node_modules/@lit-labs/context/index.d.ts:6:71 - error TS2307: Cannot find module './lib/context-request-event.js' or its corresponding type declarations.

6 export { ContextCallback, ContextRequestEvent as ContextEvent, } from './lib/context-request-event.js';
                                                                        ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

--- a/.changeset/metal-swans-decide.md
+++ b/.changeset/metal-swans-decide.md
@@ -1,0 +1,5 @@
+---
+'@lit-labs/context': patch
+---
+
+Fixes files list for package to include lib files.

--- a/packages/labs/context/package.json
+++ b/packages/labs/context/package.json
@@ -25,6 +25,7 @@
   "files": [
     "/src/",
     "!/src/test/",
+    "/lib/",
     "/development/",
     "!/development/test/",
     "/index.{d.ts,d.ts.map,js,js.map}"

